cli/client.go

```diff
@@ -215,6 +215,34 @@ func (c *viamClient) getBillingConfig(cCtx *cli.Context, orgID string) error {
 	return nil
 }
 
+// OrganizationDisableBillingServiceAction corresponds to `organizations billing disable`.
+func OrganizationDisableBillingServiceAction(cCtx *cli.Context) error {
+	c, err := newViamClient(cCtx)
+	if err != nil {
+		return err
+	}
+	orgID := cCtx.String(generalFlagOrgID)
+	if orgID == "" {
+		return errors.New("cannot disable billing service without an organization ID")
+	}
+	return c.organizationDisableBillingServiceAction(cCtx, orgID)
+}
+
+func (c *viamClient) organizationDisableBillingServiceAction(cCtx *cli.Context, orgID string) error {
```

Comment:
[super nit] make variable more informative - like `client`

Reply:
same as above - It seems like this is used elsewhere in the code base so I guess it is ok for the sake of being consistent

Reply:
yeah I can't change this to make it more verbose without changing all to the viamClient -- think this is fine for now

Reply:
I totally agree